moztelemetry/dataset.py

```diff
@@ -156,10 +156,8 @@ def summaries(self, limit=None):
 
         with futures.ProcessPoolExecutor(MAX_CONCURRENCY) as executor:
             scanned = self._scan(schema, [self.prefix], clauses, executor)
```

Comment:
How much of summaries is spent in scanning the directories vs listing the keys usually?